components/esp_lvgl_port/src/lvgl9/esp_lvgl_port_button.c

```diff
@@ -1,5 +1,5 @@
 /*
- * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
```

Comment:
does the lvgl8 port need similar modifications?

Reply:
Yes, it does. I will do it. This PR is not finished yet.

Reply:
Now, the PR is finished and ready for review :-)